comps/comps-foreman-rhel7.xml

```diff
@@ -34,6 +34,7 @@
       <packagereq type="default">foreman-service</packagereq>
       <packagereq type="default">foreman-telemetry</packagereq>
       <packagereq type="default">foreman-vmware</packagereq>
+      <packagereq type="default">nodejs-optimize-css-assets-webpack-plugin</packagereq>
```

Comment:
```suggestion
      <packagereq type="default">tfm-nodejs-optimize-css-assets-webpack-plugin</packagereq>
```

and then this needs proper sorting :)

Reply:
and `nodejs-optimize-css-assets-webpack-plugin` (without the `tfm-` prefix) needs adding to `comps-foreman-el8.xml` (the script currently fails to do that for two comps files :()